tools/ds-identify

```diff
@@ -281,6 +280,50 @@ read_fs_info() {
     DI_ISO9660_DEVS="${isodevs#,}"
 }
 
+read_geom() {
```

Comment:
you can rename this function to 'read_fs_info_freebsd' and the linux one 'read_fs_info_linux'.